fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/dashboard',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    plugins: [],
+    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    moduleFederation: {
+        shared: [
+            {
+                'react-router-dom': {
+                    singleton: true,
+                    import: false,
+                    version: '^6.3.0',
+                },
```

Comment:
We can grab the version definition from the package.json, then we don't need to manually update it here, if we don't forget. 

We should also add further version requirements like here: https://github.com/RedHatInsights/compliance-frontend/blob/master/fec.config.js#L23-L28

Reply:
Wooooooooow! I am adding this! :D thanks